client/request/commit.go

```diff
@@ -32,11 +32,12 @@ type CommitSelect struct {
 	// belonging to the given document.
 	DocID immutable.Option[string]
 
-	// FieldID is an optional filter which when provided will limit commits to those
+	// FieldName is an optional filter which when provided will limit commits to those
 	// belonging to the given field.
 	//
-	// `C` may be provided for document-level (composite) commits.
-	FieldID immutable.Option[string]
+	// `_C` may be provided for document-level (composite) commits. An explicit empty
+	// value will return branchable collection commits only.
+	FieldName immutable.Option[string]
```

Comment:
suggestion: Since we are introducing a name for a system level value (the composite), I think it would be more intuitive for users to also have a name for the branchable collection as well. It would be more consistant. `_B` would be fine I think. Otherwise why would ove have a name and the other be `nil`.

Reply:
Oh that's a good spot.  The composite name is not new though, it existed as `C` on the old fieldID, so I'm going to leave this as-is for now, but we should figure out what we want long term here at some point (the PR description suggests a `compositeOnly` input param, we could also have a `collectionOnly`, or perhaps an enum `type` with values `All` (default), `Composite`, `Collection`, `Field`, etc).

Thanks for the idea :)